cmd/extract.go

```diff
@@ -37,9 +37,12 @@ func newExtractCmd(rootConfig *RootCommandConfig) *cobra.Command {
 
 	var extractCmd = &cobra.Command{
 		Use:   "extract",
-		Short: "Extract the stack and your Appsody project to a local directory",
-		Long: `This copies the full project, stack plus app, into a local directory
-in preparation to build the final container image.`,
+		Short: "Extract your Appsody project to a local directory.",
+		Long: `Extract the full application (the stack and your Appsody project), into a local directory, in preparation to build the final container image.
```

Comment:
in preparation to build? that wording doesn't read very easily. Please can you work with @doveye on this wording and the wording for the flag '--target-dir'?

The 'build' flag if it's going to reference docker needs to refer to Docker with a capital D.

Reply:
You don't need the comma after the second bracket. I don't understand the "preparation to build" bit - is it explaining why you would want to extract a project? The docs say the build command does an extract anyway, so why would you want to do one separately?

Reply:
@doveye I included it there to give some context about when Appsody will use this. I will remove it from here as we say it in the `build` command. Thanks